keras_cv/callbacks/pycoco_callback_test.py

```diff
@@ -34,9 +34,11 @@ def test_model_fit_retinanet(self):
         model = keras_cv.models.RetinaNet(
             classes=10,
             bounding_box_format="xywh",
-            backbone=keras_cv.models.ResNet50V2(
-                include_top=False, include_rescaling=True, weights=None
-            ).as_backbone(),
+            backbone=keras_cv.models.ResNetV2Backbone.from_preset(
+                "resnet50_v2_imagenet",
+                load_weights=False,
+                include_rescaling=True,
+            ).extract_features(),
```

Comment:
can be a discussion for outside of this PR but do we want to support the passing of arbitrary levels/layers here?

Reply:
The body of this function is identical to `as_backbone`. If we want to enhance that method it should probably be a separate PR.

Reply:
+1, sounds good